web_src/less/_base.less

```diff
@@ -2416,7 +2416,6 @@ a.ui.basic.label:hover {
   position: relative;
   background: var(--color-box-header);
   border-color: var(--color-secondary);
-
```

Comment:
unnecessary changed line , it could be reverted.

Reply:
> unnecessary changed line , it could be reverted.

reverted